### PR TITLE
Update `README.md` to describe SociaLite

### LINKED ISSUE #14: Update `README.md` and remove AB-3 related information

### PR DESCRIPTION
**Pull request by bnjmnt4n**
Closes #14.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,9 @@
-[![CI Status](https://github.com/se-edu/addressbook-level3/workflows/Java%20CI/badge.svg)](https://github.com/se-edu/addressbook-level3/actions)
+[![CI Status](https://github.com/AY2122S1-CS2103T-F11-4/tp/workflows/Java%20CI/badge.svg)](https://github.com/AY2122S1-CS2103T-F11-4/tp/actions)
 
 ![Ui](docs/images/Ui.png)
 
+**SociaLite** is a project developed for the CS2103T Software Engineering module by [our team](docs/AboutUs.md), based off [AddressBook Level 3](https://github.com/se-edu/addressbook-level3).
+
 * This is **a sample project for Software Engineering (SE) students**.<br>
```
Comment (bnjmnt4n) [inline README.md:7]: Not sure if I should remove this paragraph: is it possibly still relevant for context? I can reword it more to focus on our application instead off Address Book though.

Comment (david-eom) [reply README.md:7]: Yep the current paragraph sounds slightly off. Perhaps we can remove the paragraph for the time being and add it back once we finish our preliminary features?

**Review by david-eom — COMMENTED** [review]
Hi Ben, I agree with you that we should reword the entire paragraph, or simply remove it for the time being. TBH this and #15 are duplicate work, but this dummy issue is just to make sure everyone's effort gets reflected on the tp dashboard.

**Review by stanley-1 — APPROVED** [review]
Modifications are in line with the specifications. Links in document lead to the correct webpages. PR is correctly linked to Milestone v1.1 and tagged to the corresponding Issue (#14).

**Pull request merged**